helsingborg-stad/app-mitt-helsingborg
# Fix/text replace hidden

I've added support for replacing text in value on an hidden field.

## Explain your solution
Added a new 'middleware' like function to find out all hidden field and run text replacer helper on that field value if input type is 'hidden'. This is done in the Form component to make the replaced value stay in place when we submit the form.

## How to test

Concrete example:

1. Checkout this branch
2. Use a form with hidden field in it and set its value to "Some text #month-1"
3. Fire upp the simulator by running the command `yarn ios`
4. Verify that the answer for the hidden field is saved with "Some text <month name>"


## Tested environments

- [] Building the Application on a Android device/simulator.
